src/APICalls.js

```diff
@@ -0,0 +1,465 @@
+          authProvider: { credentials: { username: "${username}", password: "${password}" } }}) {
+          user {
+            id
+            image
+            email
+            username
+            firstName
+            lastName
+            street
+            city
+            state
+            zip
+            createdAt
+            updatedAt
+          }
+        }
+      }
+      `
+    })
+  })
+  .then(response => response.json())
+  .then(response => console.log(response.data))
+  .catch(error => console.log(error))
+}
+
+export const userSignIn = (username, password) => {
+  return fetch("http://localhost:8000/graphql", {
+    method: "POST",
+    headers: {"Content-Type": "application/json"},
+    body: JSON.stringify( { query:`
+      mutation {
+        userSignIn(input: {credentials: {username: "${username}", password: "${password}" }}) {
+          token
+          user {
+            id
+            email
+            image
+            username
+            firstName
+            lastName
+            street
+            city
+            state
+            zip
+            createdAt
+            updatedAt
+          }
+        }
+      }
+      `
+    })
+  })
+  .then(response => response.json())
+  .then(response => console.log(response))
+  .catch(error => console.log(error))
+}
+
+export const updateUser = (id, firstName, lastName, email, street, city, state, zip, image, username) => {
+  return fetch("http://localhost:8000/graphql", {
+    method: "POST",
+    headers: {"Content-Type": "application/json"},
+    body: JSON.stringify( { query:`
+      mutation {
+        updateUser(input: { params: {
+          id: ${id}
+          image: ${image}
+          email: ${email}
+          firstName: ${firstName}
+          lastName: ${lastName}
+          state: ${state}
+          city: ${city}
+          zip: ${zip}
+          street: ${street}
+          username: ${username}
+        }}) {
+          user {
+            id
+            image
+            email
+            username
+            firstName
+            lastName
+            street
+            city
+            state
+            zip
+            createdAt
+            updatedAt
+          }
+        }
+      }
+      `
+    })
+  })
+  .then(response => response.json())
+  .then(response => console.log(response))
+  .catch(error => console.log(error))
+}
+
+export const createRecipe = (userId, title, description, instructions, charityId, charityName, ingredients) => {
+  return fetch("http://localhost:8000/graphql", {
+    method: "POST",
+    headers: {"Content-Type": "application/json"},
+    body: JSON.stringify( { query:`
+    mutation {
+      createRecipe(input: {params: {
+          userId: ${userId}, 
+          title: "${title}", 
+          description: "${description}", 
+          instructions: "${instructions}", 
+          charityId: ${charityId}, 
+          charityName: "${charityName}", 
+          ingredients: "${ingredients}"}}) {
```

Comment:
This is the only code that's currently broken. I'll have to talk to Hash about how ingredients need to be passed

Reply:
Ingredients have been a sticking point for a little while now, so that doesn't surprise me.